crates/parry2d/tests/geometry/epa_convergence.rs

```diff
@@ -0,0 +1,35 @@
+use na::Vector2;
+use parry2d::{
+    math::{Isometry, Point, Real, Vector},
+    query,
+    shape::{Capsule, ConvexPolygon},
+};
+
+#[test]
+fn capsule_convergence() {
+    let p = Point::from(Vector::y() * 10.0);
+
+    let shape1 = Capsule::new(-p, p, 10.0);
+    let mut vec = Vec::<Point<Real>>::with_capacity(3);
+    vec.push(Point::<Real>::new(64.0, 507.0));
+    vec.push(Point::<Real>::new(440.0, 326.0));
+    vec.push(Point::<Real>::new(1072.0, 507.0));
+    let shape2 = ConvexPolygon::from_convex_polyline(vec);
+    let shape2 = shape2.unwrap();
+    let transform1 = Isometry::new(Vector2::new(381.592, 348.491), 0.0);
+    let transform2 = Isometry::new(Vector2::new(0.0, 0.0), 0.0);
+
+    let res = query::details::contact_support_map_support_map(
+        &transform1.inv_mul(&transform2),
+        &shape1,
+        &shape2,
+        10.0,
+    )
+    .expect("Penetration not found.");
+
+    if let Ok(Some(contact)) = query::contact(&transform1, &shape1, &transform2, &shape2, 1.0) {
+        panic!("collision");
```

Comment:
The execution panis there, which seems surprising 🤔  ; same behaviour is observed on parry master.

Reply:
Interesting, and you sure you tried the exact same test as I put in that repo?

Reply:
I copied information from the branch, but I'm not ruling out an error on my end, as I'm not confident exactly what I'm reproducing.

It being os related would be an interesting information ; I'll try to reproduce on your original reproduction and report.

Reply:
If you can reproduce it, try checking my logs of what im getting inside epa2, and see what logs you have inside epa2.

Reply:
The parameters I pass aren't exactly the same:

<details><summary>Original bug</summary>
<p>

```txt
(
    Isometry {
        rotation: Complex {
            re: 1.0,
            im: 0.0,
        },
        translation: [
            381.592,
            348.491,
        ],
    },
    SharedShape ( Arc<Capsule(Capsule { segment: Segment { a: [-0.0, -5.0], b: [0.0, 5.0] }, radius: 10.0 })> ),
    Isometry {
        rotation: Complex {
            re: 1.0,
            im: 0.0,
        },
        translation: [
            0.0,
            0.0,
        ],
    },
    SharedShape ( Arc<ConvexPolygon(ConvexPolygon { points: [[64.0, 507.0], [440.0, 326.0], [1072.0, 507.0]], normals: [[[-0.4337436, -0.9010364]], [[0.27532378, -0.9613516]], [[0.0, 1.0]]] })> ),
    1.0,
)
```

</p>
</details> 

<details><summary>This pull request as of 249cca8c29c91ec39bcd6159171b5a983b272fa1</summary>
<p>

```txt
(
    Isometry {
        rotation: Complex {
            re: 1.0,
            im: 0.0,
        },
        translation: [
            381.592,
            348.491,
        ],
    },
    Capsule {
        segment: Segment {
            a: [
                -0.0,
                -5.0,
            ],
            b: [
                0.0,
                5.0,
            ],
        },
        radius: 10.0,
    },
    Isometry {
        rotation: Complex {
            re: 1.0,
            im: 0.0,
        },
        translation: [
            0.0,
            0.0,
        ],
    },
    ConvexPolygon {
        points: [
            [
                64.0,
                507.0,
            ],
            [
                440.0,
                326.0,
            ],
            [
                1072.0,
                507.0,
            ],
        ],
        normals: [
            [
                [
                    -0.4337436,
                    -0.9010364,
                ],
            ],
            [
                [
                    0.27532378,
                    -0.9613516,
                ],
            ],
            [
                [
                    0.0,
                    1.0,
                ],
            ],
        ],
    },
    1.0,
)
```

</p>
</details>

Reply:
I updated the test to be correct :

I confirm that on parry/master, this test fails, and passes with this pull request.